napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -88,6 +90,8 @@ def changeBlending(self, text):
             Name of blending mode, eg: 'translucent', 'additive', 'opaque'.
         """
         self.layer.blending = self.blendComboBox.currentData()
+        # GL minimum blending does not support changing alpha
+        self.opacitySlider.setVisible(self.layer.blending != 'minimum')
```

Comment:
Why doesn't minimum blending work with alpha?

Reply:
This also seems like a relatively easy test to write (or to add to an existing test changing blending modes).

Reply:
> Why doesn't minimum blending work with alpha?
> 
> (This is a good UI solution given this statement; I'm just missing as to why it's needed. :)

GL_MIN blending does not use a source or destination factors in the blend equation. See:
https://registry.khronos.org/OpenGL-Refpages/gl4/html/glBlendEquation.xhtml

>This also seems like a relatively easy test to write (or to add to an existing test changing blending modes).

I'm not great at tests, what should be tested?
You mean adding `minimum` here:
https://github.com/napari/napari/blob/65aef46073c430a576dc0537b0db6ab89dfcff20/napari/layers/image/_tests/test_image.py#L294-L308

Reply:
I was thinking of a test to make sure that `opacitySlider.visible` is set correctly to match the blending mode. `test_image.py` has nothing GUI in it.

A quick search suggests that
https://github.com/napari/napari/blob/main/napari/_qt/_tests/test_qt_viewer.py might be the right location.

Reply:
Ok, that makes sense. 
Basically thinking out loud:
make a viewer, generated a np.ones or zeros image, add it, check for opacity slider (should be there by default), switch to `minimum`, recheck for opacity slider (should be hidden).
Sound OK?
I guess `minimum` should be added to the test I linked too though.

Reply:
Derp, I already added the simple test to `test_image`.
Now I've added to `test_qt_image_base_controls` just seems to fit better than `test_qt_viewer`